litmus-portal/frontend/src/views/GetStarted/PasswordSet/index.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable react/no-danger */
+import { TextField, Typography } from '@material-ui/core';
+import { ButtonFilled } from 'litmus-ui';
+import React, { useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { validateConfirmPassword } from '../../../utils/validate';
+import useStyles from './styles';
+
+interface PasswordReset {
+  password: string;
+  confirmPassword: string;
+}
+
+interface PasswordSetProps {
+  handleNext: () => void;
+  currentStep: number;
+  totalStep: number;
+  setPassword: (e: any) => void;
+}
+const PasswordSet: React.FC<PasswordSetProps> = ({
+  handleNext,
+  currentStep,
+  totalStep,
+  setPassword,
+}) => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  const [values, setValues] = React.useState<PasswordReset>({
+    password: '',
+    confirmPassword: '',
+  });
+  const isError = useRef(true);
+  const isSuccess = useRef(false);
+  if (
+    values.password.length > 0 &&
+    values.confirmPassword.length > 0 &&
+    validateConfirmPassword(values.password, values.confirmPassword) === false
+  ) {
+    isError.current = false;
+    isSuccess.current = true;
+  } else {
+    isError.current = true;
+    isSuccess.current = false;
+  }
+
+  const handleSubmit = () => {
+    handleNext();
+  };
+
+  return (
+    <div className={classes.rootDiv}>
+      <div className={classes.rootLitmusText}>
+        <img src="icons/LitmusLogoLight.svg" alt="litmus logo" />
+        {/* TODO: Add translations */}
+        <Typography className={classes.HeaderText}>
+          Set your new password
+        </Typography>
+        <Typography className={classes.litmusText}>
+          The password should be of 8 characters and should not be same as the
+          password shared by admin.
```

Comment:
We don't have any password validation, we can replace it with this text instead: "Set a new password which will be used when you login next time."